### PR TITLE
Code coverage check and publish coverage to kusto

### PR DESCRIPTION
**Pull request by pjohnst5**
**Reason for Change**:
Adds a code coverage check to our builds, and publishes the code coverage percentage to kusto

### CODE REVIEW

File: .pipelines/pipeline.yaml [hunk 1]
```diff
@@ -128,90 +128,138 @@ stages:
...
-            os: linux
-            name: npm
-      steps:
-      - template: containers/container-template.yaml
-        parameters:
-          name: $(name)
-          os: $(os)
-          arch: $(arch)
-          tag: $(TAG)
+      - job: containerize
```
Comment (rbtr) [inline .pipelines/pipeline.yaml:189]: is this whitespace change from some autoformatter? i think lists should be dedented to the same level as their parent item for max readability

**Comment by rbtr** [comment]
@pjohnst5 does this have a different intent from https://github.com/Azure/azure-container-networking/pull/1222? if not, we should keep using the same PR for the same conceptual change

File: .pipelines/pipeline.yaml (hunk 1 above)
Comment (pjohnst5) [reply .pipelines/pipeline.yaml:189]: Yeah it was, I can change it

**Comment by pjohnst5** [comment]
@rbtr  yes it is actually, I'll switch to #1222

**Comment by pjohnst5** [comment]
Moving to #1222

**Pull request closed without merging**